src/main/webapp/style/tb.css

```diff
@@ -83,7 +83,7 @@ tr.secondarycolor {
 	background-color: #c1d5eb;
 }
 
-/*-- T�tigkeitsbericht --*/
+/*-- T�tigkeitsbericht --*/
```

Comment:
Warum wird das Encoding hier von UTF8 weggeändert?

Reply:
Der Salat nutzt nicht UTF-8 in seinen Sourcen, sondern cp1252 und iso8859-1. Das müssen wir noch zu UTF-8 konsequent ändern. Kann ich aber reverten.

Reply:
Muss nicht reverted werden.